app/src/main/res/layout/pocket_video_item.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<android.support.v7.widget.CardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="12dp"
+    android:layout_marginEnd="12dp"
+    app:cardCornerRadius="4dp"
+    app:cardElevation="0dp"
+    android:focusable="true"
+    >
+
+    <LinearLayout
+        android:layout_width="220dp"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        >
+
+        <ImageView
+            android:id="@+id/videoThumbnailView"
+            android:layout_width="match_parent"
+            android:layout_height="124dp"
+            tools:background="#e00"
+            tools:ignore="ContentDescription" />
```

Comment:
Also worth thinking about at some point, what accessibility strings will we need for these new features? Probably best to get an early LAT out (even if it is all hard-coded) so eitan can take a look (or we can even do a sanity pass).